Solutions/Digital Shadows/Playbooks/DigitalShadowsPlaybookToUpdateSeverity.json

```diff
@@ -0,0 +1,406 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "",
+        "description":  "",
+        "prerequisites":  "",
+        "prerequisitesDeployTemplateFile":  "",
+        "lastUpdateTime":  "",
+        "entities":  [
+        ],
+        "tags":  [
+        ],
+        "support":  {
+            "tier":  "community",
+            "armtemplate":  "Generated from https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator"
+        },
+        "author":  {
+            "name":  ""
```

Comment:
All of these metadata fields need filling in.

Reply:
Filled the metadata fields. Can you please check if the entities and tags field makes sense, I have filled it as per my understanding about the SearchLight portal

Reply:
See new comments - I'll close this one